regression-tests/sparktkregtests/testcases/frames/frame_timeseries_test.py

```diff
@@ -98,5 +100,74 @@ def test_frame_timeseries_dickey_fuller_constant_trend_squared(self):
         self.assertAlmostEqual(result.p_value, df_ctt_result[1], delta=0.0001)
         self.assertAlmostEqual(result.test_stat, df_ctt_result[0], delta=0.01)
 
+    def test_frame_timeseries_breusch_pagan(self):
+        """Test Breusch Pagan"""
+        dataset = self.get_file("breusch.csv")
+        schema = [("time", float),
+                  ("heteroskedastic", float),
+                  ("sine", float),
+                  ("uniform", float)]
+
+        frame = self.context.frame.import_csv(
+            dataset, delimiter= ' ', header=True, schema=schema)
+        heteroskedastic_result = frame.timeseries_breusch_pagan_test("heteroskedastic", "time")
+        uniform_result = frame.timeseries_breusch_pagan_test("uniform", "time")
+
+        times = frame.take(frame.count(), columns=['time'])
+        heteroskedastic_column = frame.take(frame.count(), columns="heteroskedastic")
+        heteroskedastic_residual = [item**2 for sublist in heteroskedastic_column for item in sublist]
+        reg1 = linear_model.LinearRegression()
+        heteroskedasticOLS = reg1.fit(times, heteroskedastic_residual)
+        heteroskedastic_stat = heteroskedasticOLS.score(times, heteroskedastic_residual) * frame.count()
+
+        uniform_column = frame.take(frame.count(), columns="uniform")
+        uniform_residual = [item**2 for sublist in uniform_column for item in sublist]
+        reg2 = linear_model.LinearRegression()
+        uniformOLS = reg2.fit(times, uniform_residual)
+        uniform_stat = uniformOLS.score(times, uniform_residual) * frame.count()
+
+        self.assertLess(heteroskedastic_result.p_value, 0.05)
+        self.assertAlmostEqual(heteroskedastic_result.test_stat, heteroskedastic_stat)
+        self.assertGreater(uniform_result.p_value, 0.05)
+        self.assertAlmostEqual(uniform_result.test_stat, uniform_stat)
+
+    def test_frame_timeseries_breusch_godfrey(self):
+        """Test Breusch Godfrey"""
+        dataset = self.get_file("breusch.csv")
+        schema = [("time", float),
+                  ("heteroskedastic", float),
+                  ("sine", float),
+                  ("uniform", float)]
+        max_lag=1
+        frame = self.context.frame.import_csv(
+            dataset, delimiter= ' ', header=True, schema=schema)
+        sine_result = frame.timeseries_breusch_godfrey_test("sine", ['time'], max_lag)
+        uniform_result = frame.timeseries_breusch_godfrey_test("uniform", ['time'], max_lag)
+
+        time_data = frame.take(frame.count(), columns=['time'])
```

Comment:
Can you break these 2 blocks up like you did in breusch pagan? That is, associate all the sine downloads/regression together and all the uniform downloads/regression together?

Thanks.